README.md

```diff
@@ -62,7 +62,7 @@ Deep equality as defined in user libraries can vary significantly, in part due t
 #### `Record`
 
 ```js
-const document = #{
+const record = #{
```

Comment:
Could we call this variable `proposal`? I don't really like `record` because it doesn't describe the _contents_ of the variable 😬

Reply:
@nicolo-ribaudo yes, i agree